File: beecrowd_exercicios_em_Python/begginner/ddd.py
def find_ddd(a, b):

    if b in a:
        print(a[b])

    return "DDD nao cadastrado"


def check_ddd(x):
    ddd = {61: 'Brasilia', 71: 'Salvador', 11: 'Sao Paulo', 21: 'Rio de Janeiro', 32: 'Juiz de Fora', 19: 'Campinas',
           27: 'Vitoria', 31: 'Belo Horizonte'}

    return find_ddd(ddd, x)


n = int(input())

print(check_ddd(n))

File: beecrowd_exercicios_em_Cpp/beginner/ddd.cpp
#include <iostream>
#include <map>


using namespace std;

void buscar_ddd(int n){
    int ddd [] {61,71,11,21,32,19,27,31};
    string dest [] = {"Brasilia", "Salvador", "Sao Paulo", "Rio de Janeiro",
                      "Juiz de Fora","Campinas", "Vitoria","Belo Horizonte"};

    for(int i = 0; i < 8; i ++) {
        if (n == ddd[i]){
            cout << dest[i] << endl;
            return;
        }
    }
    cout << "DDD nao cadastrado" << endl;
}

int main(){

    int n;
    cin >> n;

    buscar_ddd(n);

    return 0;
}